webdriver/tests/bidi/browsing_context/navigate/invalid.py

```diff
@@ -28,7 +28,7 @@ async def test_params_url_invalid_type(bidi_session, new_tab, value):
         )
 
 
-@pytest.mark.parametrize("value", ["http://:invalid", "http://#invalid"])
+@pytest.mark.parametrize("value", ["https://:invalid", "https://#invalid"])
```

Comment:
This doesn't really matter here. It'd be nice to add `http` as well here.

Reply:
Sure. Can add another parameterized value for protocol.